pkg/collector/corechecks/oracle-dbm/sql_wrappers.go

```diff
@@ -72,6 +76,26 @@ func isConnectionError(err error) bool {
 	return false
 }
 
+func isConnectionRefused(err error) bool {
+	return strings.Contains(err.Error(), "connect: connection refused")
+}
+
+func handleRefusedConnection(c *Check, db *sqlx.DB, err error) (bool, error) {
+	if err == nil {
+		return false, err
+	}
+	if isConnectionRefused(err) {
+		closeDatabase(c, db)
+		return true, fmt.Errorf(`%w
+The network connection between the Agent and the database server is disrupted. Run one of the following commands on the machine where the Agent is running to test the network connection: 
+nc -v dbserver port
+telnet dbserver port
+curl dbserver:port`,
```

Comment:
@alexandre-normand wondering if you think its worth hooking this up to agent warnings 

that way we could create a troubleshooting doc for oracle, [similar to sqlserver](https://docs.datadoghq.com/database_monitoring/setup_sql_server/troubleshooting/#common-connection-issues), which might look nicer than printing these commands in the agent logs

Reply:
I created the ticket https://datadoghq.atlassian.net/browse/DBMON-2796 to do that. Let me know your final decision.